src/components/UserProfileSettings/UserProfile.tsx

```diff
@@ -8,13 +8,25 @@ import styles from './UserProfileSettings.module.css';
 interface InterfaceUserProfile {
   firstName: string;
   lastName: string;
+  createdAt: string;
   email: string;
   image: string;
 }
+const prettyDate = (param: string): string => {
```

Comment:
Can it be `joinedDate` to indicate function behaviour?

Reply:
Yes Sure